### PR TITLE
restrict #pods/node to 50 for now

### PR DESCRIPTION
**Pull request by jim-minter**
```release-note
restrict #pods/node to 50 as a starting point for GA
```

### CODE REVIEW

File: pkg/vmimage/data/etc/origin/node/bootstrap-node-config.yaml [hunk 1]
```diff
@@ -20,22 +20,26 @@ imageConfig:
...
+  feature-gates:
+  - RotateKubeletClientCertificate=true,RotateKubeletServerCertificate=true
+  max-pods:
+  - "50"
+  pod-manifest-path:
+  - /etc/origin/node/pods
+  rotate-certificates:
+  - "true"
+  runtime-request-timeout:
+  - 10m
```
Comment (asalkeld) [inline pkg/vmimage/data/etc/origin/node/bootstrap-node-config.yaml]: this is new, what does it do?

Comment (mjudeikis) [reply pkg/vmimage/data/etc/origin/node/bootstrap-node-config.yaml]: How long kubelet will try to GET,PUT, to `docker` to get stuff out. Does not include watchers (like logs).

**Pull request merged**